README.md

```diff
@@ -35,7 +35,7 @@ If you are a partner, you are welcome to add your logo/link to our partners page
 This site uses [Jekyll](https://jekyllrb.com/). You can build the site and make it available on a local server via `docker-compose up -d`, or by installing all the dependencies on your local environment as follows.
 
 1. Install [Ruby](https://www.ruby-lang.org/en/) and [Bundler](https://bundler.io/), then run `bundle install`.
-2. Build and start Jekyll with `bundle exec jekyll serve`.
+2. Build and start Jekyll with `bundle exec jekyll serve`. **NOTE** To test blogs with a future publication date use `bundle exec jekyll serve --future`
```

Comment:
As per my comment on #280 can we be less prescriptive here and link to the Jekyll docs? Teach a man to fish and all that.

I'd rather keep the number instructions clean, so I'd be in favour of putting this below the number instructions and without the all caps/bold "NOTE."

Reply:
https://jekyllrb.com/docs/configuration/options/

Reply:
adjusted accordingly